src/components/DisplayComments.jsx

```diff
@@ -0,0 +1,27 @@
+import { useState, useEffect } from "react";
+import { getComments } from "../utils/api";
+
+const DisplayComments = ({ selectedArticle }) => {
+  const [comments, setComments] = useState([]);
+  useEffect(() => {
+    getComments(selectedArticle.article_id).then((comments) => {
+      setComments(comments);
+    });
+  }, [selectedArticle]);
+  return (
+    <>
+      <h3>comments</h3>
+
+      {comments.map((comment) => {
+        return (
+          <>
```

Comment:
Don't forget that React needs a `key` prop on the element that gets rendered from a `.map`. A fragment might not be the best choice here.